source/unified-test-format/unified-test-format.rst

```diff
@@ -0,0 +1,2515 @@
+    YAML file SHOULD use an `alias node`_ for this value (e.g.
+    ``databaseName: *database0Name``).
+
+.. _expectedEvent_commandSucceededEvent:
+
+- ``commandSucceededEvent``: Optional document. Assertions for a one or more
+  `CommandSucceededEvent <../command-monitoring/command-monitoring.rst#api>`__
+  fields.
+
+  The structure of this document is as follows:
+
+  - ``reply``: Optional document. Test runners MUST follow the rules in
+    `Evaluating Matches`_ when processing this assertion.
+
+  - ``commandName``: Optional string. Test runners MUST assert that the actual
+    command name matches this value using a case-insensitive comparison.
+
+.. _expectedEvent_commandFailedEvent:
+
+- ``commandFailedEvent``: Optional document. Assertions for a one or more
+  `CommandFailedEvent <../command-monitoring/command-monitoring.rst#api>`__
+  fields.
+
+  The structure of this document is as follows:
+
+  - ``commandName``: Optional string. Test runners MUST assert that the actual
+    command name matches this value using a case-insensitive comparison.
+
+
+collectionOrDatabaseOptions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Map of parameters used to construct a collection or database object.
+
+The structure of this document is as follows:
+
+  - ``readConcern``: Optional document. See `commonOptions_readConcern`_.
+
+  - ``readPreference``: Optional document. See `commonOptions_readPreference`_.
+
+  - ``writeConcern``: Optional document. See `commonOptions_writeConcern`_.
+
+
+Common Options
+--------------
+
+This section defines the structure of common options that are referenced from
+various contexts in the test format. Comprehensive documentation for some of
+these types and their parameters may be found in the following specifications:
+
+- `Read and Write Concern <../read-write-concern/read-write-concern.rst>`__.
+- `Server Selection: Read Preference <../server-selection/server-selection.rst#read-preference>`__.
+
+The structure of these common options is as follows:
+
+.. _commonOptions_collectionName:
+
+- ``collectionName``: String. Collection name. The YAML file SHOULD use an
+  `alias node`_ for a collection entity's ``collectionName`` field (e.g.
+  ``collectionName: *collection0Name``).
+
+.. _commonOptions_databaseName:
+
+- ``databaseName``: String. Database name. The YAML file SHOULD use an
+  `alias node`_ for a database entity's ``databaseName`` field (e.g.
+  ``databaseName: *database0Name``).
+
+.. _commonOptions_readConcern:
+
+- ``readConcern``: Document. Map of parameters to construct a read concern.
+
+  The structure of this document is as follows:
+
+  - ``level``: Required string.
+
+.. _commonOptions_readPreference:
+
+- ``readPreference``: Document. Map of parameters to construct a read
+  preference.
+
+  The structure of this document is as follows:
+
+  - ``mode``: Required string.
+
+  - ``tagSets``: Optional array of documents.
+
+  - ``maxStalenessSeconds``: Optional integer.
+
+  - ``hedge``: Optional document.
```

Comment:
This one too? `{ enabled: boolean }` [doc](https://docs.mongodb.com/manual/reference/method/cursor.readPref/#cursor-readpref-tagset:~:text=Optional.%20A%20document%20that%20specifies%20whether%20to%20enable%20the%20use%20of%20hedged%20reads.)

Reply:
Never noticed this but shell calls it `hedgeOptions` 👀 opportunity for consistency?

Reply:
`hedge` corresponds to the term in the [Server Selection spec](https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#hedge). I'd rather be consistent with our internal specs than anything the shell does.

Note: no spec tests presently use `hedge`, but I opted to include this for completeness since it is defined in the SS spec. Would it be preferable to just link to the specific component sections in that spec for each of these rather than duplicating explanations, or is the single link to the SS spec at the top of this section sufficient?